### DISCUSSION TITLE
Q about copy filtered files

### DISCUSSION THREAD

**Discussion by cblpop** [WinMerge/winmerge#779] [Q&A]
Hi, this program is awesome, but seems im dumb or thats a bug?
I have 2 folders 1 folder is full and 2 folder have few(near hundred for exmple) deleted files, my mission is to define which files were deleted and there was no problem with it, but i also need to copy files (which were deleted) from folder 1 to folder №3 and theres kind of problem...
I filtered (from view tab) to ONLY show files that i need, but when i need click folder and click "copy/from left to (and select folder №3)" it copies WHOLE folder, but not files that only shows in compare when i filtered it...
i found that selecting each individual file is work, but theres pretty much folders so i need to open every folder from list and select all individual files that is a huge waste of time :)

So the question is kind of is there any way to just click ATRL+A from the **filtered** files list and click "copy to" and it would copy only files that i filtered?

much ty :)

**Comment by sdottaka** [comment]
It may not be the answer you want,
you can easily select the files to copy by unchecking the View→Tree Mode menu item to set it to Flat Mode.

![image](https://user-images.githubusercontent.com/98126/117222675-fb2e6e80-ae46-11eb-9b96-663574be13bf.png)
